sql/core/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -533,6 +533,12 @@ object SQLConf {
       .timeConf(TimeUnit.MILLISECONDS)
       .createWithDefault(10L)
 
+  val WHOLETEXT =
+    SQLConfigBuilder("spark.sql.wholetext")
```

Comment:
Should this really be a session-global configuration? It seems like something that is specific to a particular input file and should only be set when opening a given file.